### DISCUSSION TITLE
[Not an issue] A message to say thanks and showcase what I did with no prior Jekyll or Ruby experience in a few days

### DISCUSSION THREAD

**Discussion by aexarch** [alshedivat/al-folio#623] [Show and tell]
Greetings,

First things first, I would like to thank you for making an excellent Jekyll theme for academics.

My brother had a pretty lame github.io site that wasn't responsive and was super basic so I looked into static site generators, then themes and stumbled upon your awesome project.

My innovations while creating my brother's academic portfolio site:

1. no computers or Ruby installations were used on my end. I adapted the materials from his old site, wrote custom Ruby scripts and migrated the repo through an rm rf all from my phone through code editors, a Git client on Android and the magic that is Github Actions (CI).

2. Instead of settling for Android webbrowser's official lack of inline pdf viewing, I utilized Mozilla's PDF.js library via the magic of Liquid Tags and Ruby plugin support to use it in the project. While this can probably be done more intelligently via utilizing a CDN instead of self-hosting the library, it works.

3. For things that did not work properly when switching from a development (project) repo to a user site (different source branch, baseurl/project folder suffix not being applied when pointing to root subdirectories, I was able to make an almost identical codebase to work in both cases, all through looking things up and studying your code. I also discovered how great kramdown is. I managed to turn links into bootstrap/mdb buttons using only standard markdown link code followed by 
```
{: .classes}
```

The site up to now only took over existing content, no diagrams or mathjax formulas were utilized yet, but I learned how to import and extend .bib files, batch exported from Google Scholar.

I would love it if you were to take a look at the result and want to include it in your example site list or have any feedback for me.

The site can be viewed at https://iexarchos.github.io and my brother's repo is located at: https://github.com/iexarchos/iexarchos.github.io

Though the development and testing took place at https://github.com/aexarch/iexarchos.github.io until recently.

Thanks for reading through this wall of text and once again for all of your hard work and this great theme. Truely eye candy.

**Comment by alshedivat** [comment]
thanks for sharing your experience with the community! since this is not an issue, converting it into a discussion.
